### PR TITLE
Blog: Add article about adjust the volume to "how-to-invite-bot"

### PR DESCRIPTION
**Pull request by NathanTW0219**

### CODE REVIEW

File: blog/2022-05-07-how-to-invite-bot.mdx [hunk 1]
```diff
@@ -115,6 +115,14 @@ YEE式機器龍是我正在製作的一個項目，他可以讓你播放YouTube
 | 繼續播放               | &resume                                        |
 | 中斷機器人的音樂           | &leave                                         |
 | 讓機器人加入語音頻道         | &join                                          |
-| 隨機打亂播放清單           | &shuffle                                       |
+| 隨機打亂播放清單           | &shuffle             <br/>                          |
```
Comment (Gary50613) [inline blog/2022-05-07-how-to-invite-bot.mdx:118]: 為什麼這裡有一個換行?

**Review by Gary50613 — CHANGES_REQUESTED** [review]
圖片請用 https://www.iloveimg.com/convert-to-jpg
轉成jpeg之後調整大小到 128x128
不然會拖慢載入速度

File: blog/2022-05-07-how-to-invite-bot.mdx (hunk 1 above)
Comment (NathanTW0219) [reply blog/2022-05-07-how-to-invite-bot.mdx]: Idk, Maybe GitHub copilot wants to add it and I accidentally pressed the tab

**Review by Gary50613 — APPROVED** [review]
LGTM!

**Pull request merged**